torch/onnx/symbolic_opset11.py

```diff
@@ -881,6 +881,24 @@ def prim_ConstantChunk(g, self, chunks, dim):
         start = end
     return res
 
+def chunk(g, self, chunks, dim):
+    # Calculate chunk size for dynamic chunk
+    dim_size = g.op("Gather", g.op("Shape", self), dim, axis_i=0)
+    chunk_size_s = g.op("Sub", chunks, g.op("Constant", value_t=torch.tensor([1], dtype=torch.long)))
```

Comment:
why we need chunks-1 rather than chunks?

Reply:
The formula to calculate chunk_size from chunk in the exporter is:
chunk_size = (dim_size + chunks - 1) / chunks
That's why the -1

Reply:
[1] dtype = chunks dtype?